airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -604,7 +607,7 @@ def get_client_type(
         self,
         region_name: str | None = None,
         config: Config | None = None,
-        deferrable: bool = False,
+        deferrable: bool = DEFAULT_DEFERRABLE,
```

Comment:
I don't think setting `deferrable` to `True` will work here, nor is it necessary. `deferrable` should only be set `True` by getting the `async_conn` property, which calls the series of methods with `deferrable=True`. 

Even in an operator using deferrable mode, we want the ability to create non-deferrable sessions because we often make synchronous API calls before going to deferrable mode.

Reply:
Got it. Just change it back to `False`. Thanks for your reviewing!